lib/assessment.rb

```diff
@@ -20,20 +30,35 @@
 # Name and location should be writeable, but age should be read-only.
 
 ##
-# your code here
+class Person
+  attr_writer :name
+  attr_writer :location
```

Comment:
Missed this earlier, but these should really be `attr_accessor`, not `attr_writer`. `attr_writer` creates a 'setter' method but doesn't create a 'getter' one. Because it's very unusual for someone to be able to edit a value but not be able to see it, `attr_writer` is rarely used.